src/express-middleware.js

```diff
@@ -57,6 +57,11 @@ class ExpressMiddleware {
                 route = route ? route + req.route.path : req.route.path;
             }
 
+            // #112 - aggregated express default route
```

Comment:
There are three cases we need to worry about skipping for the default route.

1. Routine route assignment (ln65)
2. Appending when includeQueryParams is true (ln76)
3. Nest.js case is actually replacing `/hello*` with `0*` (ln82)

Because this affected so many branches, I opted for a short-circuit. Feedback and suggestions welcome.

Reply:
as it seems we're missing support `*` in general.
according to `express` documentation
>This route path will match abcd, abxcd, abRANDOMcd, ab123cd, and so on.
```js
app.get('/ab*cd', (req, res) => {
  res.send('ab*cd')
})
```
and also regex support, which we can ignore for now.

shouldn't we support all routes with `*` instead?
the main issue here is that we'd need to also support `*` with request params. i.e
```js
app.get('/ab*cd/:id', (req, res) => {
  res.send('ab*cd')
})
```
this isn't a must for now, but would have been a more complete solution.

lmk what you think.

Reply:
I'd be interested to pursue the general solution.  If I can get an answer to [this question](https://github.com/PayU/prometheus-api-metrics/issues/114#issuecomment-1612577291), I'll have a better understanding of the requirements and can take a stab at that.